### PR TITLE
Update the DEVELOPMENT guide to reference Node 18 LTS and npm 9

### LINKED ISSUE #92: Update the DEVELOPMENT guide to reference Node 18 LTS and npm 9
At the moment the DEVELOPMENT guide says we suggest using Node 16 and npm 8, when we should really be targeting Node LTS 18 and npm 9.

See here:
https://github.com/JamesLMilner/terra-draw/blob/main/guides/DEVELOPMENT.md

Fix would be to update this document and also ensure that the package-lock.json is update for npm 9. To achieve this just do npm install with npm 9 installed and checkin the lockfile if it changes.

### PR DESCRIPTION
**Pull request by jaiakash**
Fixes #92

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,236 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.0.2](https://github.com/JamesLMilner/terra-draw/compare/v0.0.1...v0.0.2) (2023-10-01)
```
Comment (JamesLMilner) [inline CHANGELOG.md:5]: We don't need to do a release or update the CHANGELOG for this change. I'll show you how we can undo this!

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
-<img src="./logo.png" alt="Terra Draw Logo" width="400"/>
+<img src="./readme.gif" alt="Terra Draw Logo" width="400"/>
```
Comment (JamesLMilner) [inline README.md:1]: We don't want to change the logo here I don't think. Can we undo this change?

**Review by JamesLMilner — COMMENTED** [review]
Thanks for the PR @jaiakash.  A few things here: 

* We don't need to do a release here so we will need to undo the changes in the CHANGELOG and changing the package.json version number.
* There were grammatical and spelling changes to the README which is helpful, although for some reason the logo was changed?
* To keep our change log clean, we'll need to get this into one nice commit message like `docs: updated DEVELOPMENT to reference Node 18`. These means we'll need to change the commit history. The easiest thing to do here is probably to do `git reset HEAD~5` and then just commit the README with the logo reversion and the DEVELOPMENT.md file without the other changes.

Thanks again for your PR! Look forward to getting this in after these revisions.

File: README.md (hunk 2 above)
Comment (jaiakash) [reply README.md:1]: Sure

**Comment by jaiakash** [comment]
Hi, Sorry for unnecessary changes. Thanks for the explanation. Have done the changes according to that.

**Comment by JamesLMilner** [comment]
@jaiakash no problem! I'm glad the explanation was helpful :) This looks great and I can merge - thank you for your helpful contribution.
